en/ch04.md

```diff
@@ -7,14 +7,12 @@ The `Send` tab provides tools for creating outgoing transactions.
 (2) **Transaction priority:** This is the priority level your transaction will receive in the pool of transactions waiting to be confirmed. The more you pay, the higher your transactions priority for inclusion in a block.  
 *Currently staying with the default or the slow option is likely to get you into the next block.*  
 (3) **Address:** This is where you put the Monero address that you are sending to. Best practice is to copy and paste the address to prevent errors, accompanied with visually checking that the pasted address is correct.  
-(4) **Payment ID (optional):** The payment ID is an identifier attached to the transaction you are about to send. Often when sending to an exchange they will give you a payment ID that you must include here. This is so they know which incoming transaction is from you.  
-*If you forget to add your payment ID you should still be able to recover your funds by contacting the party you sent Monero to.*  
-*Long payment IDs are being deprecated. It is hidden by default. If you need to specify a long payment ID, you should enable it first on the [Settings > Layout](#layout) tab. You should encourage services to accept payments that need to be identified with unique subaddresses instead*  
-(5) **Description (optional):** This is for your record keeping. You can add some information regarding your transaction for future reference.   
-(6) **Sweep unmixable:** This allows you to get rid of outputs in your wallet which have strange amounts like `0.000006839355`. These are unmovable without combining them with another output.  
+(4) **Description (optional):** This is for your record keeping. You can add some information regarding your transaction for future reference. The description is not sent to the receiver (it is only seen by you).  
+(5) **Sweep unmixable:** This allows you to get rid of outputs in your wallet which have strange amounts like `0.000006839355`. These are unmovable without combining them with another output.  
 *Most users will never need to use this feature.*  
-(7) **Sign tx file:** This button allows you to sign a transaction file that was created on a view-only wallet.  
-(8) **Export key images:** This option will allow you to export your key images. They are needed to get the true balance of a view-only wallet.
+(6) **Sign tx file:** This button allows you to sign a transaction file that was created on a view-only wallet.  
+(7) **Export key images:** This option will allow you to export your key images. They are needed to get the true balance of a view-only wallet.  
+(8) **Import key images:** This option will allow you to import key images into your view-only wallet. When your view-only wallet has the key images for all outputs, it can display the true balance. In order to see this button in your Send page, you must open a view-only wallet and use a local node.
```

Comment:
```suggestion
(8) **Import key images (advanced):** This option will allow you to import key images into your view-only wallet. When your view-only wallet has the key images for all outputs, it can display the true balance. In order to see this button in your Send page, you must open a view-only wallet and use a local node.
```

Probably better to specify, since 'simple mode' users won't see the button.